src/js/components/molecules/content-block/content-block.scss

```diff
@@ -1,5 +1,9 @@
 .content-block {
     @extend %reset-last-margin;
+
+    &__description {
+        margin-bottom: 24px;
```

Comment:
Zou je dit willen veranderen in $buildingUnit * 2

Reply:
Done